src/OpenTelemetry/Metrics/MeterProviderBuilderBase.cs

```diff
@@ -29,10 +29,14 @@ namespace OpenTelemetry.Metrics
     /// </summary>
     public abstract class MeterProviderBuilderBase : MeterProviderBuilder
     {
+        internal const int MaxMetrics = 1000;
```

Comment:
Remove this variable since the name is misleading (it is not the max, it's just the default of the limit).